### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewIntegratedTerminal_ScreenReader: Jaws does not read the information that displays in the editor.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52270]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: Jaws 18.0.5038

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Integrated Terminal.
3. Tab till the editor.

**Actual:**
 Jaws does not read the information that displays in the editor(Microsoft Windows Version and copyright part) and also messages that display after execution of command, those are not read by JAWS.

**Expected:**
JAWS should read each and every information present in the editor. And when user enters any command or changes the command by using tab/arrow keys that should be read by screenreader and also the message that displays after the execution.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know the result that displays after execution of command.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Terminal Jaws not reading editor's content.pptx](https://github.com/Microsoft/vscode/files/2114211/A11y.VSCode.Terminal.Jaws.not.reading.editor.s.content.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by auchenberg** [comment]
Closing this issue as it's only observed in JAWS, and this isn't strictly needed to meet MAS C compliance.
